src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -1046,11 +1046,11 @@ paths:
       description: |
         **Note:** this endpoint is available in Alfresco 7.4 and newer versions.
 
-        Assign the node **nodeId** to a category. You specify the category id in a JSON body like this:
+        Assign the node **nodeId** to a category. You specify the category ID in a JSON body like this:
 
         ```JSON
         {
-          "categoryId": "01234567-89ab-cdef-0123-456789abcdef"
+          "id": "01234567-89ab-cdef-0123-456789abcdef"
```

Comment:
`categoryId` is the correct name. We shouldn't use `id` in this case because the endpoint is `.../category-links` not `categories`

Reply:
Exactly as Maciej mentioned. If we will leave here `categoryId` in input then we should fix output class `Category` to return `categoryId` instead `id`.

Reply:
After re-thinking - it might be possible to have `categoryId` on input and `id` in output by adding setter `setCategoryId(id)` to `Category` class 🤔